study.md

```diff
@@ -66,7 +66,7 @@ Which ActiveRecord method finds all of the records of a certain type (or
 entity)?
 
 ```md
-<!-- your response here -->
+.find_by()
```

Comment:
`#find_by` retrieves at most one record.  You would want to use `#all` in this case.